cpp/include/cudf/io/detail/json.hpp

```diff
@@ -51,4 +51,9 @@ void write_json(data_sink* sink,
                 json_writer_options const& options,
                 rmm::cuda_stream_view stream,
                 rmm::mr::device_memory_resource* mr);
+
+rmm::device_uvector<char> normalize_single_quotes(const cudf::device_span<std::byte>& inbuf,
```

Comment:
No need to pass a span by reference. 
Also, marking a span object (or a reference) as const does not make the content const. Here's how a const span should be passed:
```suggestion
rmm::device_uvector<char> normalize_single_quotes(cudf::device_span<std::byte const> inbuf,
```

Reply:
Thanks, fixed!